### PR TITLE
filters added

### PR DESCRIPTION
**Pull request by ashifkhn**
Latest deploy log | https://app.netlify.com/sites/d-gadget-store2/deploys/624e93ae91b4690009aedf50
-- | --
Deploy Preview | https://deploy-preview-6--d-gadget-store2.netlify.app/

### CODE REVIEW

File: src/pages/ProductsPage/productspage.css [hunk 1]
```diff
@@ -21,6 +21,7 @@ categories {
   justify-content: center;
   align-items: center;
   flex-grow: 1;
+  /* position: absolute; */
 }
```
Comment (vikrantkode) [inline src/pages/ProductsPage/productspage.css:25]: remove commented code

**Review by vikrantkode — APPROVED** [review]
- remove commented code
- good to go

File: src/backend/db/products.js [hunk 2]
```diff
@@ -12,47 +12,177 @@ export const products = [
     image: "https://images.unsplash.com/photo-1625842268584-8f3296236761?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80",
     price: "50000",
     categoryName: "PC",
+    rating: 4,
+    fastDelivery: false,
+    inStock: true
   },
   {
     _id: uuid(),
```
Comment (pnchinmay) [inline src/backend/db/products.js:20]: Keep the indentation consistent

File: src/components/Sidebar.jsx [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+import { CartState } from "../context/Context";
 import "./Style/sidebar.css";
 
 export const Sidebar = () => {
+  const {
+    productDispatch,
+    productState: { byStock, byFastDelivery, sort, byRating },
+  } = CartState();
+
+  // console.log(byStock, sort, byRating, byFastDelivery);
```
Comment (pnchinmay) [inline src/components/Sidebar.jsx:11]: Remove these commented out code

**Review by pnchinmay — APPROVED** [review]
Left a few sanity comments

File: src/pages/ProductsPage/productspage.css (hunk 1 above)
Comment (ashifkhn) [reply src/pages/ProductsPage/productspage.css:25]: thank you. Will refactor.

File: src/backend/db/products.js (hunk 2 above)
Comment (ashifkhn) [reply src/backend/db/products.js:20]: I'll refactor.

**Pull request merged**